pkg/maps/policymap/policymap.go

```diff
@@ -89,6 +89,19 @@ type PolicyEntryDump struct {
 	Key PolicyKey
 }
 
+// PolicyEntriesDump is a wrapper for a slice of PolicyEntryDump
+type PolicyEntriesDump []PolicyEntryDump
+
+// Less returns true if the element in index `i` is lower than the element
```

Comment:
This comment isn't helpful, as that is the generic definition of what `Less` is for anything which implements the Sort interface. Update it to specify that it sorts by traffic direction, and then by identity.